Makefile

```diff
@@ -66,6 +66,26 @@ build-storybook: install generate-api-types  ## Builds the storybook.
 
 # MISC
 
+.PHONY: scraper
+scraper: ## Inserts the scrapers scraped data into the database.
+	cd scraper && npm install && mkdir -p data && npm run start:full
+
+.PHONY: migration
+migration: ## Database migration.
+	cd backend && make migration
+
+.PHONY: migration-redo
+migration-redo: ## Runs the down.sql and then the up.sql for the most recent migration.
+	cd backend && make migration-redo
+
+.PHONY: migration-redo-a
+migration-redo-a: ## Runs the down.sql and then the up.sql for all migration.
+	cd backend && make migration-redo-a
+
+.PHONY: migration-reset
+migration-reset:  ## Reset diesel database.
```

Comment:
Is a bit dangerous at it obscures a bit what it does: resetting the whole database.